baseframe/forms/auto.py

```diff
@@ -18,9 +18,12 @@ class ConfirmDeleteForm(Form):
     cancel = SubmitField(__(u"Cancel"))
 
 
-def render_form(form, title, message='', formid='form', submit=__(u"Submit"), cancel_url=None, ajax=False, with_chrome=True):
+def render_form(form, title, message='', formid='', submit=__(u"Submit"), cancel_url=None, ajax=False, with_chrome=True):
     multipart = False
-    ref_id = 'form-' + buid()
+    if not formid:
+        ref_id = 'form-' + buid()
+    else:
+        ref_id = 'form-' + formid
```

Comment:
This could be `ref_id = 'form-' + (formid or buid())`

Reply:
Fixed